workspace_hack/Cargo.toml

```diff
@@ -24,6 +24,7 @@ fail = { version = "0.5", default-features = false, features = ["failpoints"] }
 futures-channel = { version = "0.3", features = ["alloc", "futures-sink", "sink", "std"] }
 futures-task = { version = "0.3", default-features = false, features = ["alloc", "std"] }
 futures-util = { version = "0.3", features = ["alloc", "async-await", "async-await-macro", "channel", "futures-channel", "futures-io", "futures-macro", "futures-sink", "io", "memchr", "sink", "slab", "std"] }
+getrandom = { version = "0.2", default-features = false, features = ["js", "js-sys", "rdrand", "std", "wasm-bindgen"] }
```

Comment:
`js`, `js-sys`, `wasm-bindgen` look weird for our use-case

Reply:
Yeah, they come through `flume`. Testing it out in the next push to leave them behind once more.

Reply:
Kept the `flume` for mpmc queue as I got the #2880 reimplemented in the latest commits.